src/app/components/ATIAnalytics/params/index.test.ts

```diff
@@ -487,6 +509,64 @@ describe('ATIAnalytics params', () => {
       });
     });
 
+    it('should return the correct Homepage params', () => {
+      const homePageData = {
+        pageData: {
+          metadata: homePage,
+          pageType: 'home',
+          title: 'pageTitle',
+          curations: [
+            {
+              summaries: [
+                {
+                  type: 'article',
+                  title:
+                    '“Баланы сабады деп аялымды камап салышарын өзүм да билген эмесмин”. Кадамжайда токмоктолгон наристенин атасы 3\r',
+                  firstPublished: '2023-02-02T10:06:57.156Z',
+                  link: 'https://www.bbc.com/kyrgyz/articles/cemg3359nwro',
+                  imageUrl:
+                    'https://ichef.bbci.co.uk/ace/standard/{width}/cpsdevpb/2105/test/a7436f40-a2dd-11ed-9015-6935ab4fa6ca.jpg',
+                  description: '',
+                  imageAlt: 'test',
+                  id: 'cemg3359nwro',
+                },
+              ],
+              activePage: 1,
+              pageCount: 1,
+              curationId:
+                'urn:bbc:tipo:list:524cf34b-cac2-4ce2-ac64-53eb70019202',
+              curationType: 'tipo-curation',
+              position: 1,
+              title: 'Редактордун тандоосу',
+              visualProminence: 'HIGH',
+              visualStyle: 'COLLECTION',
+            },
+          ],
+          description: 'Hello I am a description!',
+        },
```

Comment:
I am not sure if we need all of this - what's the bare minimum we can get away with?

Reply:
Addressed this now in the latest PR changes here:
https://github.com/bbc/simorgh/pull/10899